src/modules/auth/components/login/loginForm/loginForm.tsx

```diff
@@ -28,7 +32,12 @@ export function LoginForm() {
   }
 
   function onSubmit(data: LoginFormData) {
-    console.log(data);
+    axiosInstance
+      .post('/auth/login', data)
+      .then(() => navigate('/'))
```

Comment:
are we not storing tokens in cookie/localStorage?

Reply:
I'm currently working on refresh token logic. There I will update the endpoints to set the cookies instead of returning the access/refresh token in json.